MBBSEmu.Tests/ExportedModules/Majorbbs/movmem_Tests.cs

```diff
@@ -33,12 +32,50 @@ public void movmem_Test(ushort moveLength, byte[] expected)
                     srcPointer.Segment,
                     dstPointer.Offset,
                     dstPointer.Segment,
+                    (ushort) expected.Length
+                });
+
+            //Verify Results
+            var dstArray = mbbsEmuMemoryCore.GetArray("DST", (ushort) expected.Length);
+
+            Assert.Equal(0, mbbsEmuCpuRegisters.AX);
+            Assert.Equal(0, mbbsEmuCpuRegisters.DX);
```

Comment:
this function turned out to be `void` right? I believe I was mistaken when I said it should return a value. If `void`, just don't assert on the return value at all.

Reply:
Yes this is void